themes/V3/Blank/style.less

```diff
@@ -12,7 +12,7 @@
 }
 
 @page { margin : 0; }
-body { counter-reset : page-numbers; }
+body { counter-reset : page-numbers 0; }
```

Comment:
Default is 0 anyway, consider this change has no effect.

Reply:
Preference for explicitness.